workers/unity/Packages/com.improbable.gdk.core/Worker/ConnectionHandlers/ConnectionFlowInitializers.cs

```diff
@@ -0,0 +1,99 @@
+using Improbable.Worker.CInterop;
+
+namespace Improbable.Gdk.Core
+{
+    // TODO: Should this be contravarient?
```

Comment:
keeping the TODO in?

Reply:
Yes as I'm not entirely sure (need a use case) for doing so